reports/pharmacy_first_report.Rmd

```diff
@@ -767,5 +768,116 @@ plot_measures(
   y_label = "Number of codes for PF conditions"
 ) + scale_color_manual(values = ethnicity_palette)
 ```
+```{r, message=FALSE, warning=FALSE, echo = FALSE}
+df_bsa_validation <- read_csv(here("lib", "validation", "data", "pf_consultation_validation_data.csv"))
```

Comment:
I can see why you started loading the data here, ideally we load this data at the top of the script, where we also load the other data.